build/Repositories.props

```diff
@@ -1,55 +1,55 @@
-    <Repository Include="JsonPatch" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="KestrelHttpServer" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Localization" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Logging" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="MetaPackages" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)">
-      <Branch>rel/2.0.1</Branch>
-    </Repository>
-    <Repository Include="Microsoft.Data.Sqlite" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="MusicStore" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Mvc" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="MvcPrecompilation" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Options" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Proxy" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Razor" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="ResponseCaching" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Routing" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Scaffolding" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Security" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="ServerTests" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Session" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="StaticFiles" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="Testing" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
-    <Repository Include="WebSockets" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
+    <!-- Repos being patched -->
+    <Repository Include="EntityFrameworkCore" Branch="rel/2.0.1" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
+    <Repository Include="Mvc" Branch="rel/2.0.1" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
+    <Repository Include="Identity" Branch="rel/2.0.1" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
+    <Repository Include="MetaPackages" Branch="rel/2.0.1" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
+
+    <!-- TODO compute downstream repos that should be updated. -->
+
+    <!--
+    Repos not building this patch.
+    Build tools will *verify* that these repos will be unaffected
+    by the patch update and do not need updating.
+    -->
+    <VerifyRepository Include="Antiforgery" Branch="rel/2.0.0" Commit="" CloneUrl="git@github.com:aspnet/%(Identity)" />
```

Comment:
We still need the list of packages that these produced so we can build a runtime store with the right set of packages.